Gemfile

```diff
@@ -15,6 +15,7 @@ group :development do
   gem 'better_errors'
   gem 'binding_of_caller'
   gem 'quiet_assets'
+  gem 'railroady'
```

Comment:
Sweet. These are great to use. I haven't tried this one. I've been using [rails-erd](https://github.com/voormedia/rails-erd) up til now.